server/lib/db/transaction.ctrl.js

```diff
@@ -29,7 +29,7 @@ TransactionCtrl.internal.createTransaction = (req, res) => {
     TransactionCtrl.runtime = { req, res }
 
     TransactionCtrl.runtime.transaction = new TransactionModel({
-      changelly_id: req.params.changellyId,
+      changelly_id: req.params.changellyId || '123123123',
```

Comment:
why is this defaulting to a static value out of interest?

Reply:
We currently have no way to get a changelly id so this simplifies things for testing.
Perhaps I should add a comment in the code above it explaining?

Reply:
thats fine, just wanted to know why it was here.